packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -96,6 +97,7 @@ export declare namespace CreateRelease {
     };
     body: {
       name: string;
+      scheduledAt?: Date;
```

Comment:
Don't know the extent of the PR, asking in just in case :) 

can you create a release without a scheduled at value?  Is it set to a default value if not setting it?
Shouldn't you be able to update that scheduledAt value when updating a release?

Reply:
Hey @Marc-Roig, yes, you can create a release without a scheduledAt value, by default I guess we keep null just to know that is not scheduled.

And yes, you would be able to update the scheduledAt date when updating a release but in an upcoming PR 🙌 

I started with this one because I think is the most important thing to unblock all the others tasks, but probably the one about editing is going to be the next one 😅

Reply:
nice! I am liking the incremental and small PRs, its so much easier to review